src/player.py

```diff
@@ -1,2 +1,48 @@
 # Write a class to hold player information, e.g. what room they are in
 # currently.
+
+
+class Player:
+    def __init__(self, name, current_room):
+        self.name = name
+        self.current_room = current_room
+        self.inventory = []
+
+    def getCurrentRoom(self):
+        return self.current_room
+
+    def setCurrentRoom(self, current_room):
+        self.current_room = current_room
+
+    def checkInventory(self):
+        print("You carry with you: ")
+        for item in self.inventory:
+            print(item)
+
+    def followPath(self, path):
```

Comment:
Not immediately clear what this method is doing. Could be helpful to include docstring or comments